VERSION

```diff
@@ -94,28 +94,28 @@ date="Unreleased developer copy"
 # Version numbers are described in the Libtool current:revision:age
 # format.
 
-libmpi_so_version=80:0:0
-libmpi_mpifh_so_version=80:0:0
-libmpi_usempi_tkr_so_version=80:0:0
-libmpi_usempi_ignore_tkr_so_version=80:0:0
-libmpi_usempif08_so_version=80:0:0
-libopen_pal_so_version=80:0:0
-libmpi_java_so_version=80:0:0
-liboshmem_so_version=80:0:0
-libompitrace_so_version=80:0:0
+libmpi_so_version=80:0:40
+libmpi_mpifh_so_version=80:0:40
+libmpi_usempi_tkr_so_version=80:0:40
+libmpi_usempi_ignore_tkr_so_version=80:0:40
+libmpi_usempif08_so_version=80:0:40
+libopen_pal_so_version=80:0:40
```

Comment:
The openpal interface changed in a backwards incompatible way.  It should be reset to 0.  Applications only link against libmpi, so this shouldn't matter for ABI compliance issues.

Reply:
Sure.  Done.